compiler/util/src/org/jetbrains/kotlin/config/LanguageVersionSettings.kt

```diff
@@ -264,6 +264,7 @@ enum class LanguageFeature(
     RangeUntilOperator(KOTLIN_1_8), // KT-15613
     GenericInlineClassParameter(sinceVersion = KOTLIN_1_8, kind = UNSTABLE_FEATURE), // KT-32162
     DataObjects(KOTLIN_1_8), // KT-4107
+    AssignOperatorOverloadForJvm(KOTLIN_1_8),
```

Comment:
I think the feature should be disabled by default. Please move it to experimental group (and remove KOTLIN_1_8).

Reply:
Moved to experimental group, thank you.